src/store/record.js

```diff
@@ -122,6 +126,22 @@ let recordStore = {
             state.sections.dataAccess.changes = 0;
             state.sections.dataAccess.message = "Record successfully updated!";
         },
+        setAdditionalInformation(state, additionalInformation) {
+            let record = {
+                access_points: additionalInformation['access_points'],
+            };
+            // TODO: Separate setting necessary for each available field...
+            ['access_points'].forEach((type) => {
+                state.sections.generalInformation.data.metadata[type] = JSON.parse(JSON.stringify(record[type]));
+                state.sections.generalInformation.initialData.metadata[type] = JSON.parse(JSON.stringify(record[type]));
+                state.sections.dataAccess.data.metadata[type] = JSON.parse(JSON.stringify(record[type]));
+                state.sections.dataAccess.initialData.metadata[type] = JSON.parse(JSON.stringify(record[type]));
```

Comment:
I don't think this is needed. DataAccess will take the extra metadata it needs from generalInformation.initialData. So you are setting something that isn't used.
See `updateDataAccess()`: https://github.com/FAIRsharing/fairsharing.github.io/blob/1ed1bf3052348b321c801c9ed98971b39f9d2e9e/src/store/record.js#L352-L354